src/cli_validator/loader/cmd_meta/__init__.py

```diff
@@ -38,43 +36,48 @@ def load_blob_text(url: str, cache_path: Optional[str] = None, cache_strategy: C
     return data
 
 
-def load_version_index(target_dir: Optional[str] = None):
-    cache_path = f'{target_dir}/version_list.txt' if target_dir else None
-    data = load_blob_text(f'{BLOB_URL}/{CONTAINER_NAME}/version_list.txt', cache_path, CacheStrategy.Fallback)
+def load_version_index(target_dir: Optional[str] = None, ext_name: Optional[str] = None):
+    ext_sep = f'/azure-cli-extensions/ext-{ext_name}' if ext_name else ''
+    cache_path = f'{target_dir}{ext_sep}/version_list.txt' if target_dir else None
+    data = load_http(f'{BLOB_URL}/{CONTAINER_NAME}{ext_sep}/version_list.txt', cache_path, CacheStrategy.Fallback)
     data = data.strip(' \n')
-    return [v.strip()[10:] for v in data.split() if v.strip()]
+    return [v.strip() for v in data.split() if v.strip()]
```

Comment:
Why is `[10:]` no longer needed?

Reply:
The method extract only the version number previously. However, we now require it returning the full entry name like `azure-cli-2.61.0` since we reuse this method when handling extension version index.